lib/utils/routes.js

```diff
@@ -44,6 +45,10 @@ function routes(server) {
     res.end('</body></html>');
 
     function writeDirectory(baseUrl, basePath) {
+      if (isWebpack5 && basePath === 'auto') {
+        basePath = '';
+      }
```

Comment:
I'm not sure how to handle the `auto` public path. Is there a way to retrieve it?